### ISSUE TITLE
Media Key not received on windows.

### ISSUE THREAD

**Issue by didrip** [headsetapp/headset-electron#237]
I just installed latest headset on windows 10 (1809) and I cannot get the media key to work.

I check #5 and googled for solutions (below) but still have the problem.

- change default Music & Video app => I set it to VLC which receives the media key events correctly (it did so even before I set it as the default app)

- disable google music extension

- chrome://flags/ => disable hardware handling media keys

- disable Windows Media Player Network Sharing Service

- quit any other media application, and/or restart windows

edit : works fine on same computer on ubuntu 18.10 with same headset version

**Comment by fcastilloec** [comment]
I've been trying to reproduce this on the exact same version of Windows you have but I can't.

Media keys in Windows work as first-come-first-served, which means that if you have an app already running that uses media keys, that app will receive all of those media keys. If the app only uses a couple of media keys then the rest can still be grabbed by Headset. For example, Chrome can grab the `play-pause` key when you're playing media on a tab. You can still change songs on Headset with media keys but you won't be able to pause them, Chrome will pause the video/audio playing in its window.

I don't believe this is a bug, but just in case we're going to need some logs. Make sure Headset is not running, then go to `C:\Users\$USER\AppData\Roaming\Headset` where `$USER` is your windows' user name. Delete the file `log.log`.
Make sure that no other apps are running, not even a browser, and then start Headset, play a song and try using the three supported media keys (`play-pause`, `play-next`, `play-previous`). Close headset and send the newly created `log.log` file so I can see what's happening.

**Comment by ydixt** [comment]
Thanks for your feedback, I'll take time to investigate this week end, but I don't think it is an headset bug either anymore as I found that play/pause works from my bluetooth headset (headphones), whereas media keyboard key does not on Headset (both work on VLC). So it might be some sort of a Logitech (brand of my keyboard) setting.

**Comment by didrip** [comment]
Tried again with the updated version (3.0.2) but the error log stays desperately empty : (I tried only the play/pause key as this is the only one on my keyboard, Logitech K810).

```
2019/07/12-08:11:40.428 2d50 Recovering log #25
2019/07/12-08:11:40.461 2d50 Delete type=0 #25
2019/07/12-08:11:40.463 2d50 Delete type=3 #24
```

**Comment by didrip** [comment]
After trying to quit every single background app one by one I finally found the guilty blocking app : Logitech Setpoint Event Manager (had to kill it in task manager). Weird thing is that it did not stop media keys from being received by VLC.

**Comment by didrip** [comment]
Found solution to keep using Setpoint (useful for low battery notifications) and make media keys work as normal, all credits to this post https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/1306#issuecomment-264827884

Basicaly I used [TextCrawler](https://www.digitalvolcano.co.uk/tcdownloads.html) to find the xml file containing the name of my keyboard inside the folder C:\ProgramData\Logishrd\SetPointP\Devices\Keyboard

Then edited the XML file to comment out any <Button> block with HandlerSet="MM ..." except for the volume keys which work fine. There was only one in my case. Kill then Relaunch Setpoint and it works like a charm.
